napari/utils/colormaps/colormap_utils.py

```diff
@@ -385,7 +385,10 @@ def vispy_or_mpl_colormap(name):
     else:
         try:
             mpl_cmap = getattr(cm, name)
-            display_name = _MATPLOTLIB_COLORMAP_NAMES[name]
+            if name in _MATPLOTLIB_COLORMAP_NAMES:
+                display_name = _MATPLOTLIB_COLORMAP_NAMES[name]
+            else:
+                display_name = trans._(name)
```

Comment:
I don't think it makes sense to do `trans._` here, right @goanpeca? `trans._` only works with string literals?

Reply:
and anyway we'd need `trans._p` @DragaDoncila to specify the "colormap" context.

Reply:
>I don't think it makes sense to do trans._ here, right @goanpeca? trans._ only works with string literals?

Correct!

Reply:
```suggestion
                display_name = name
```

Reply:
Sorry, I was following the same idea as in the vispy colormaps [here](https://github.com/napari/napari/blob/336772a7d4123795dccb644d75adbb203efae360/napari/utils/colormaps/colormap_utils.py#L147). Can you explain what the difference is?

Reply:
:shrug: @goanpeca? Is that another oversight?

Reply:
Probably 🤷